### DISCUSSION TITLE
seqlevelsStyle(annotations) <- 'UCSC' and CreateChromatinAssay error

### DISCUSSION THREAD

**Discussion by yangchao4** [stuart-lab/signac#702] [General]
Hello， I am trying to analysis  RNA+ATAC  and use Signac .
when i trying to make a annotation file , code as follows
library(EnsDb.Mmusculus.v79)
annotations <- GetGRangesFromEnsDb(ensdb =EnsDb.Mmusculus.v79 )
seqlevelsStyle(annotations) <- 'UCSC'
there is a error:
**Error in .order_seqlevels(chrom_sizes[, "chrom"]) :
!anyNA(m31) is not TRUE**

I find there is discuss about this error , as link:  https://github.com/Bioconductor/GenomeInfoDb/issues/27#issuecomment-871181898
but i can not solve the error in follow-up code 
chrom_assay <- CreateChromatinAssay(
  counts = atac_counts,
  sep = c(":", "-"),
  genome = 'mm10',
  fragments = frag.file,
  min.cells = 10,
  annotation = annotations
)
there is a error too:
**Error in .order_seqlevels(chrom_sizes[, "chrom"]) :
!anyNA(m31) is not TRUE**

**Comment by timoast** [comment]
See https://github.com/timoast/signac/issues/687
